variables.tf

```diff
@@ -86,3 +86,12 @@ variable "vnet_location" {
   type        = string
   default     = null
 }
+
+variable "ddos_protection_plan" {
+  description = "The set of DDoS protection plan configuration"
+  type = object({
```

Comment:
Almost LGTM but we have a format issue here, would you please run `terraform fmt` command for this pr? Thanks

Reply:
`terraform fmt` applied to this code.In the case of object type, it seems that indentation is applied differently than expected because it is multi-line.

If you prefer the one-line expression as shown below, I will edit it and commit it in one line.
![image](https://user-images.githubusercontent.com/26956917/183009372-6280bf07-e459-47fd-9332-a1969124f3cd.png)

Reply:
Sorry for the late reply, LGMT, Thanks!